numerrin_wrapper/numerrin_mesh.py

```diff
@@ -0,0 +1,449 @@
+            uuid of the desired point.
+
+        Returns
+        -------
+        Point
+            Mesh point identified by uuid
+
+        Raises
+        ------
+        Exception
+            If the point identified by uuid was not found
+
+        """
+
+        try:
+            coords = numerrin.getnode(self.pool.ph, self.name,
+                                      self._uuidToNumLabel[uuid])
+            point = Point(coords, uuid)
+            for dataName in numvariables:
+                try:
+                    mDataName = self.name + dataName
+                    if numerrin.gettype(self.pool.ph, mDataName) == "Function":
+                        dataV = numerrin.getrealfunction(self.pool.ph,
+                                                         mDataName)
+                    else:
+                        dataV = numerrin.getvariable(self.pool.ph, mDataName)
+                    dkey = [key for key, value in numname.iteritems() if
+                            value == dataName][0]
+                    point.data[dkey] = dataV[self._uuidToNumLabel[uuid]]
+                except:
+                    pass
+
+            return point
+        except KeyError:
+            error_str = "Trying to get an non-existing point with uuid: {}"
+            raise ValueError(error_str.format(uuid))
+
+    def get_edge(self, uuid):
+        """Returns an edge with a given uuid.
+
+        Returns the edge stored in the mesh
+        identified by uuid. If such edge do not
+        exists an exception is raised.
+
+        Parameters
+        ----------
+        uuid
+            uuid of the desired edge.
+
+        Returns
+        -------
+        Edge
+            Edge identified by uuid
+
+        Raises
+        ------
+        Exception
+            If the edge identified by uuid was not found
+
+        """
+        message = "Edges are not supported yet in Numerrin engine"
+        raise NotImplementedError(message)
+
+    def get_face(self, uuid):
+        """Returns a face with a given uuid.
+
+        Returns the face stored in the mesh
+        identified by uuid. If such face do not
+        exists an exception is raised.
+
+        Parameters
+        ----------
+        uuid
+            uuid of the desired face.
+
+        Returns
+        -------
+        Face
+            Face identified by uuid
+
+        Raises
+        ------
+        Exception
+            If the face identified by uuid was not found
+
+        """
+
+        try:
+            pointLabels = self.pool.get_face_points(self.name,
+                                                    self._uuidToNumLabel[uuid])
+            puids = [self._numPointLabelToUuid[lbl] for lbl in pointLabels]
+            face = Face(puids, uuid)
+            try:
+                blabel = self.pool.get_face_boundary_label(
+                    self.name, self._uuidToNumLabel[uuid])
+                face.data[CUBA.LABEL] = blabel
+            except:
+                pass
```

Comment:
Again, not safe. Wouldn't it be safer to execute the code block without the `try-except`?